### DISCUSSION TITLE
Stream Play with grey frames

### DISCUSSION THREAD

**Discussion by pankajsaini1987** [bluenviron/mediamtx#1326] [Questions]
hi, when i receive stream from rtsp-simple-server
server works well and make connection with client as well ( client i am using vlc/ffmpeg)

but during stream play i am getting grey frame on most of the stream. what is that, how can i avoid these grey frames during play of stream.

**Comment by aler9** [comment]
There's a section in the README that describes what to do in case of corrupted streams:

https://github.com/aler9/rtsp-simple-server#corrupted-frames

[a comment by an automated account is not shown]
